docs/sphinx/reference-libobs-util.rst

```diff
@@ -8,6 +8,7 @@ Platform/Utility API Reference (libobs/util)
    reference-libobs-util-bmem
    reference-libobs-util-circlebuf
    reference-libobs-util-config-file
+   reference-libobs-util-deque
    reference-libobs-util-darray
```

Comment:
```suggestion
   reference-libobs-util-darray
   reference-libobs-util-deque
```
Alphabetization.

Reply:
No because:
- Double-ended queue
- Dynamic Array
- Dynamic String

Reply:
Are these _really_ alphabetized by their display name and not the name of the page?

Reply:
Realized that indeed those are alphabetized based on their identifiers and not the documentation name.

Reply:
If the HTML rendered order ends up causing confusion, blame me, and we'll figure out something.